File: Demo-LLP-MIP/Demo-LLP-MIP-Python/v1/isign/argument/cardbin/CardBinParams.py
class CardBinParams:
    def __init__(self):
        self.mch_id = None
        self.sub_mchid = None
        self.card_no = None

    def to_dict(self):
        return {
            "mch_id": self.mch_id,
            "sub_mchid": self.sub_mchid,
            "card_no": self.card_no
        }

File: Demo-LLP-MIP/Demo-LLP-MIP-PHP/v1/isign/argument/cardbin/CardBinParams.php
<?php


class CardBinParams
{
    public $mch_id;
    public $sub_mchid;
    public $card_no;

    public function __construct()
    {
        $this->mch_id = null;
        $this->sub_mchid = null;
        $this->card_no = null;
    }

    public function to_dict() {
        $data = [];
        foreach ($this as $key => $value) {
            if ($value !== null) {
                $data[$key] = $value;
            }
        }
        return json_encode($data);
    }
}